<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KSecurityListEditorDecl</class>
  <widget class="KDialog" name="KSecurityListEditorDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>722</width>
        <height>453</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Security List Editor</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <widget class="QWidget" name="m_layoutWidget">
      <layout class="QVBoxLayout">
        <item>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QTreeWidget" name="m_listView">
                <column>
                  <property name="text">
                    <string>ID</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Type</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string comment="name of the security">Name</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Symbol</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Market</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Currency</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Fraction</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Cash Fraction</string>
                  </property>
                  <property name="clickable">
                    <bool>true</bool>
                  </property>
                  <property name="resizable">
                    <bool>true</bool>
                  </property>
                </column>
              </widget>
            </item>
            <item>
              <layout class="QVBoxLayout">
                <item>
                  <widget class="KPushButton" name="m_editButton">
                    <property name="text">
                      <string>Edit...</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="KPushButton" name="m_deleteButton">
                    <property name="text">
                      <string>Delete...</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer3">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>110</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Vertical</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <spacer name="spacer6">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>51</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Vertical</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QCheckBox" name="m_showCurrencyButton">
            <property name="text">
              <string>Show national currencies</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="Line" name="line1">
            <property name="frameShape">
              <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
      <customwidget>
       <class>KDialog</class>
       <extends>QDialog</extends>
       <header>kdialog.h</header>
      </customwidget>
  </customwidgets>
</ui>
